frame/support/procedural/src/pallet/expand/hooks.rs

```diff
@@ -191,6 +191,7 @@ pub fn expand_hooks(def: &mut Def) -> proc_macro2::TokenStream {
 			}
 		}
 
+		#[cfg(feature = "std")]
```

Comment:
The Polkadot comp CI fails.  
Looks like [mock_msg_pallet](https://github.com/paritytech/polkadot/blob/6d612ab1fee2dcaaabd19ee12a9399d0c01a10f7/xcm/xcm-simulator/example/src/parachain.rs#L153) does not have an `std` feature and therefore tries to compile the integrity tests without it.  
In these cases we cannot run the integrity test, so the [__construct_runtime_integrity_test](https://github.com/paritytech/substrate/blob/8fb4138d73ad88a19331c30c36ba91796c5846e1/frame/support/procedural/src/construct_runtime/mod.rs#L577) should probably also be `std`-gated.  
Or we fix the `mock_msg_queue` to have a `std` feature :man_shrugging: